daemons/fenced/pacemaker-fenced.c

```diff
@@ -968,6 +996,24 @@ node_has_attr(const char *node, const char *name, const char *value)
     return (match != NULL);
 }
 
+/*!
+ * \internal
+ * \brief Check whether a node does watchdog-fencing
+ *
+ * \param[in] node    Name of node to check
+ *
+ * \return TRUE if node found in stonith_watchdog_targets
+ *         or stonith_watchdog_targets is emty indicating
```

Comment:
empty

Reply:
oops ... that was fixed locally already ... must have forgotten to save the file in the editor